tetrateci/test_1.8.sh

```diff
@@ -8,40 +8,47 @@ if [[ ${CLUSTER} == "gke" ]]; then
+  git apply tetrateci/eks-ingress.1.8.patch
+  go test -count=1 -tags=integ -timeout 30m ./tests/integration/telemetry/stats/... -run='TestStatsFilter|TestSetup|TestIstioctlMetrics|TestStatsFilter|TestWASMTcpMetric|TestWasmStatsFilter|TestMain|TestCustomizeMetrics'  -p 1 -test.v
+  go test -count=1 -tags=integ ./tests/integration/pilot/endpointslice/. -istio.test.skipVM true  -p 1 -test.v
 else
-  go test -count=1 -tags=integ -timeout 30m ./tests/integration/security/. -istio.test.skipVM true -p 1 -test.v
-  go test -count=1 -tags=integ ./tests/integration/security/sds_ingress/. -istio.test.skipVM true -p 1 -test.v
-  go test -count=1 -tags=integ ./tests/integration/security/sds_ingress_gateway/. -istio.test.skipVM true -p 1 -test.v
-  go test -count=1 -tags=integ ./tests/integration/security/sds_ingress_k8sca/. -istio.test.skipVM true -p 1 -test.v
-  go test -count=1 -tags=integ -timeout 30m ./tests/integration/pilot/ -run='TestAddToAndRemoveFromMesh|TestAllNamespaces|TestAuthZCheck|TestDescribe|TestDirectoryWithRecursion|TestDirectoryWithoutRecursion|TestEmptyCluster|TestEnsureNoMissingCRDs|TestErrorLine|TestFileAndKubeCombined|TestFileOnly|TestGateway|TestIngress|TestInvalidFileError|TestIstioctlMetrics|TestJsonInputFile|TestJsonOutput|TestKubeOnly|TestLocality|TestMirroring|TestMirroringExternalService|TestProxyConfig|TestProxyStatus|TestStatsFilter|TestTcpMetric|TestTraffic|TestValidation|TestVersion|TestWASMTcpMetric|TestWait|TestWasmStatsFilter|TestWebhook' -istio.test.skipVM true -p 1 -test.v
+  go test -count=1 -tags=integ -timeout 30m ./tests/integration/security/.  -p 1 -test.v
 
   if [[ ${CLUSTER} == "aks" ]]; then
-    go test -count=1 -tags=integ -timeout 30m -run='TestStatsFilter|TestStatsTCPFilter|TestSetup|TestIstioctlMetrics|TestTcpMetric|TestStatsFilter|TestWASMTcpMetric|TestWasmStatsFilter|TestMain|TestCustomizeMetrics' ./tests/integration/telemetry/stats/... -istio.test.skipVM true -p 1 -test.v
+    go test -count=1 -tags=integ -timeout 30m -run='TestStatsFilter|TestStatsTCPFilter|TestSetup|TestIstioctlMetrics|TestTcpMetric|TestStatsFilter|TestWASMTcpMetric|TestWasmStatsFilter|TestMain|TestCustomizeMetrics' ./tests/integration/telemetry/stats/...  -p 1 -test.v
   else
-    go test -count=1 -tags=integ ./tests/integration/telemetry/stackdriver/... -run='TestStackdriverHTTPAuditLogging|TestVMTelemetry' -istio.test.skipVM true -p 1 -test.v
-    go test -count=1 -tags=integ -timeout 30m ./tests/integration/telemetry/stats/... -istio.test.skipVM true -p 1 -test.v
+    go test -count=1 -tags=integ ./tests/integration/telemetry/stackdriver/... -run='TestStackdriverHTTPAuditLogging|TestVMTelemetry'  -p 1 -test.v
+    go test -count=1 -tags=integ -timeout 30m ./tests/integration/telemetry/stats/...  -p 1 -test.v
   fi
 fi
 
+go test -count=1 -tags=integ ./tests/integration/security/sds_ingress/.  -p 1 -test.v
```

Comment:
Can we move these test cases above as a common list?

Reply:
Sure